apis/flowcollector/v1beta1/flowcollector_types.go

```diff
@@ -832,6 +836,28 @@ type DebugConfig struct {
 	Env map[string]string `json:"env,omitempty"`
 }
 
+// `SubnetLabels` allows to define custom labels on subnets and IPs or to enable automatic labelling of recognized subnets in OpenShift.
+type SubnetLabels struct {
+	// `openShiftAutoDetect` allows, when set to `true`, to detect automatically the machines, pods and services subnets based on the
+	// OpenShift install configuration and the Cluster Network Operator configuration.
+	//+optional
+	OpenShiftAutoDetect *bool `json:"openShiftAutoDetect,omitempty"`
```

Comment:
is there a default

Reply:
keeping `nil` as a default makes it more flexible, imagine if in the future we want to enable it by default, then we'll be able to tell "if it's nil => enabled" .. which will also work for folks upgrading from a previous version.
If we set a default "false", and we later change the default, people upgrading will still have their old default